src/content/blog/ivpn-apps-on-all-platforms-to-undergo-security-audit.md

```diff
@@ -0,0 +1,21 @@
+---
+title: IVPN apps on all platforms to undergo security audit
+url: /blog/ivpn-apps-on-all-platforms-to-undergo-security-audit/
+highlighted: false
+draft: false
+authors:
+  - Nick Pestell
+categories:
+  - IVPN News
+tags:
+  - Audit
+date: 2020-12-11T10:53:00.000Z
+thumbnailImage: /images-static/uploads/audit.png
+---
+Earlier this year IVPN published results of a comprehensive [3rd party infrastructure audit](/blog/independent-security-audit-concluded/) that included VPN, web, authentication, API servers and supporting infrastructure. Following the open-sourcing of our apps in February this year, we decided to commission a comprehensive audit of IVPN apps for all platforms (Windows, macOS, Linux, iOS, Android).
+
+For this project we have engaged cure53, an independent firm who possess the necessary experience after completing numerous similar projects in the industry.  Five security auditors with expertise in each app platform will undertake the audit. It is scheduled to start in early Feb 2021 and we expect to publish the results no later than March 2021.
```

Comment:
please change Feb to February

Reply:
@johnnyburnaway 
- Changed "Feb" to "February"  
- Added "Apps" tag